docs/running-node-ansible.md

```diff
@@ -0,0 +1,214 @@
+In future, it will be possible to compile the application for a wide range of operating systems (thanks to Go language).
+
+### Install `Python`
+Python3 needs to be installed on target machine to run ansible playbooks
+```bash
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends python3
+```
+
+## Configure Nodes (local machine)
+
+### 1. Set network chain id in [`deployment/ansible/inventory/hosts.yml`]
+```yaml
+all:
+  vars:
+    chain_id: <chain-id>
+  ...
+```
+Every network must have a unique chain ID (e.g. `test-net`, `main-net` etc.)
+
+### 2. Put `genesis.json` file under specific directory (if you are not running a genesis node)
+Get or download `genesis.json` file of a network your node will be joining and put it under the following path:
+```
+deployment/persistent_chains/<chain-id>/genesis.json
+```
+where `<chain-id>` is the chain id of a network spefied in the previous step
+
+### 3. Set node type specific config parameters
+
+#### Genesis Node:
+[`deployment/ansible/inventory/hosts.yml`]
+  - Set Genesis node instance IP address or hostname
+    ```yaml
+    all:
+      ...
+      children:
+        genesis:
+          hosts:
+            <genesis node IP address or hostname>
+        ...
+        validators:
+          hosts:
+            <genesis node IP address or hostname>
+        ...
+    ```
+    You should set the same address for `validators` and `genesis` hosts because a genesis node is also a validator node 
+
+[`deployment/ansible/roles/configure/vars/validator.yml`]
+  - Set `persisten_peers` value in 
+      ```yaml
+      config:
+        p2p:
+          persistent_peers: "<node1-ID>@<node1-IP>:26656,..."
+        ...
+      ```
+[`deployment/ansible/inventory/group_vars/genesis.yaml`]
+  - Set genesis accounts:
+      ```yaml
+      accounts:
+      - name: validator-key
+        passphrase: password123
+        roles:
+          - NodeAdmin
+          - Trustee
+      ```
+    Genesis nodes should be created with at least one validator and trustee accounts
+
+#### Validator Node:
+[`deployment/ansible/inventory/hosts.yml`]
+  - Set Validator node instance IP address or hostname
+    ```yaml
+    all:
+      ...
+      children:
+        ...
+        validators:
+          hosts:
+            <validator node IP address or hostname>
+        ...
+    ```
+
+[`deployment/ansible/roles/configure/vars/validator.yml`]
+  - Set `persisten_peers` value in 
+      ```yaml
+      config:
+        p2p:
+          persistent_peers: "<node1-ID>@<node1-IP>:26656,..."
+        ...
+      ```
+  - (Optional) If you are joining a long-running network, enable `statesync`
+      ```yaml
+      config:
+        ...
+        statesync:
+          enable: true
+          rpc_servers: "http(s):<node1-IP>:26657, ..."
+          trust_height: 0
+          trust_hash: ""
+        ...
+      ```
+      refer to this [document](./running-node-in-existing-network.md) for detailed info
+
+#### Observer Nodes:
```

Comment:
I think we need to clearly describe how to setup different use cases, and how to enable/disable optional components:

- Genesis Validator + optional Sentries + optional Observers
- Validator + optional Sentries + optional Observers
- Observers only